### PR TITLE
Chewable improvements, and new snacks

### PR DESCRIPTION
**Pull request by alsoandanswer**
dreammaker_rkTdAlvMHY

  - rscdel: "Expired Halloween candy has been removed from snack machines."
  - maptweak: "The archaic lollipop and gumball machines have been removed."
  - rscadd: "Courtesy of Getmore Corporation, enjoy Carps Ahoy, Tuna snax, Ad-mints, Chewy Fruit gum, and lollipops in your nearest vending machine instead!"
  - rscadd: "Lollipops and gum come in wrappers now, due to hygenie concerns."

Chewy Fruit : 5 pack of flavored gum.
Lollipops : mostly unchanged
Carps Ahoy : Wheat derivative flavored with cocoa substitute
Ad-mints : Processed Spearmint residue

And now, a word from our sponsor, Getmore Corp.

### IC CHANGELOG

Courtesy of Getmore Corporation, enjoy Carps Ahoy, Ad-mints, Chewy Fruit gum, and lollipops in your nearest vending machine

Tired of being security and having a smoking addiction, leading to smoky breath?
Now with Ad-mints, you only need to worry about being security!
(like, seriously. I think a lot of RP conversations could be started just by giving some guy a mint)

Carps Ahoy! Capt'n Carpie's got some precious cookie dubloons to share with you!
(only for the low price of 20 credits) (Getmore Corporation does not have liability for allergic reactions. By consuming our product you adhere to the Getmore Consumers Agreement.)

After much request from the Tajaran community, we're now shipping Tuna snax, made from [REDACTED] derivative and [REDACTED] to make it more affordable!

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/game/objects/items/trash.dm [hunk 1]
```diff
@@ -12,19 +12,19 @@
 	return
 
 /obj/item/trash/koisbar
-	name = "\improper K'ois Bar Wrapper"
+	name = "\improper k'ois Bar wrapper"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/trash.dm]:
```suggestion
	name = "\improper k'ois bar wrapper"
```

File: code/game/objects/items/trash.dm [hunk 2]
```diff
@@ -137,10 +141,14 @@
...
+	name = "\improper space twinkie"
 	icon_state = "space_twinkie"
 
 /obj/item/trash/grease //used for generic plattered food. example is lasagna.
 	name = "square tray"
 	icon_state = "grease"
 	drop_sound = 'sound/items/trayhit1.ogg'
+
+/obj/item/trash/cookiesnack
+	name = "\improper Carps Ahoy!"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/trash.dm]: this is a very inconsistent name

File: code/game/objects/items/weapons/chewables.dm [hunk 3]
```diff
@@ -2,20 +2,37 @@
...
 	var/chem_volume = 0
 	var/chewtime = 0
 	var/brand
 	var/list/filling = list()
-
-obj/item/clothing/mask/chewable/New()
-	..()
+	var/wrapped = FALSE
+
+/obj/item/clothing/mask/chewable/attack_self(mob/user as mob)
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/chewables.dm]:
```suggestion
/obj/item/clothing/mask/chewable/attack_self(mob/user)
```

File: code/game/objects/items/weapons/chewables.dm [hunk 4]
```diff
@@ -2,20 +2,37 @@
...
+	if(wrapped)
+		wrapped = FALSE
+		to_chat(user, span("notice", "You unwrap the [name]."))
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/chewables.dm]:
```suggestion
		to_chat(user, span("notice", "You unwrap \the [name]."))
```

File: code/game/objects/items/weapons/chewables.dm [hunk 5]
```diff
@@ -118,42 +136,62 @@ obj/item/clothing/mask/chewable/Destroy()
...
-/obj/item/clothing/mask/chewable/candy/gum/New()
-	..()
+/obj/item/clothing/mask/chewable/candy/gum/Initialize()
+	. = ..()
 	reagents.add_reagent(pick("banana","berryjuice","grapejuice","lemonjuice","limejuice","orangejuice","watermelonjuice"),10)
 	color = reagents.get_color()
+	update_icon()
+
+/obj/item/weapon/storage/box/gum
+	name = "Chewy Fruit flavored gum"
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/chewables.dm]:
```suggestion
	name = "\improper Chewy Fruit flavored gum"
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 6]
```diff
@@ -576,6 +576,17 @@
 	nutriment_desc = list("sweetness" = 3, "cookie" = 2)
 	bitesize = 1
 
+/obj/item/weapon/reagent_containers/food/snacks/cookiesnack
+	name = "Carps Ahoy!"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: very inconsistent name, this should at least say what it is

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!review

**Pull request merged**